include/hal/library/cryptlib/cryptlib_cert.h

```diff
@@ -206,8 +206,11 @@ extern bool libspdm_x509_get_key_usage(const uint8_t *cert, size_t cert_size, si
  * @param[out]     usage       Key usage bytes.
  * @param[in, out] usage_size  Key usage buffer size in bytes.
  *
- * @retval  true
- * @retval  false
+ * @return true   If the returned usage_size == 0, it means that cert and oid are valid, but the Extended key usage is not found;
```

Comment:
`return` -> `retval`
`retval` is used for specific return values, like `true` and `false`.

Applies to all instances in this pull request.

Reply:
Thanks. I have changed the comment.